Makefile

```diff
@@ -102,15 +103,15 @@ build-virt-v2v-image:
 	bazel run virt-v2v:forklift-virt-v2v
 
 push-virt-v2v-image: build-virt-v2v-image
-	$(CONTAINER_CMD) tag virt-v2v:forklift-virt-v2v ${VIRT_V2V_IMAGE}
-	$(CONTAINER_CMD) push ${VIRT_V2V_IMAGE}
+	$(CONTAINER_CMD) tag virt-v2v:forklift-virt-v2v ${VIRT_V2V_IMAGE_COLD}
+	$(CONTAINER_CMD) push ${VIRT_V2V_IMAGE_COLD}
 
 build-virt-v2v-warm-image:
 	bazel run virt-v2v:forklift-virt-v2v-warm
 
 push-virt-v2v-warm-image: build-virt-v2v-warm-image
-	$(CONTAINER_CMD) tag virt-v2v:forklift-virt-v2v-warm ${VIRT_V2V_IMAGE}
-	$(CONTAINER_CMD) push ${VIRT_V2V_IMAGE}
+	$(CONTAINER_CMD) tag virt-v2v:forklift-virt-v2v-warm ${VIRT_V2V_IMAGE_WARM}
+	$(CONTAINER_CMD) push ${VIRT_V2V_IMAGE_WARM}
 
 build-operator-bundle-image:
 	bazel run operator:forklift-operator-bundle-image --action_env CONTROLLER_IMAGE=${CONTROLLER_IMAGE} --action_env VALIDATION_IMAGE=${VALIDATION_IMAGE} --action_env OPERATOR_IMAGE=${OPERATOR_IMAGE} --action_env VIRT_V2V_IMAGE=${VIRT_V2V_IMAGE_COLD} --action_env API_IMAGE=${API_IMAGE}
```

Comment:
```suggestion
	bazel run operator:forklift-operator-bundle-image --action_env CONTROLLER_IMAGE=${CONTROLLER_IMAGE} --action_env VALIDATION_IMAGE=${VALIDATION_IMAGE} --action_env OPERATOR_IMAGE=${OPERATOR_IMAGE} --action_env VIRT_V2V_IMAGE="${VIRT_V2V_IMAGE_COLD}|${VIRT_V2V_IMAGE_WARM}" --action_env API_IMAGE=${API_IMAGE}
```

Reply:
I didn't want to make this change before we actually have the image in Quay. If we want to change this right now I will need to modify also `hack/release-images.sh`.